uint16 DISABLED=0
uint16 ENABLED_TELEOP=1
uint16 ENABLED_AUTO=2
uint16 PERSISTENT_ENABLE_AUTO=3
uint16 PERSISTENT_ENABLE_TELEOP=4

Header header
uint16 state